contracts/src/v0.8/ccip/onRamp/OnRamp.sol

```diff
@@ -498,9 +498,11 @@ contract OnRamp is IEVM2AnyOnRampClient, ITypeAndVersion, OwnerIsCreator {
   }
 
   /// @notice Withdraws the outstanding fee token balances to the fee aggregator.
-  /// @dev This function can be permissionless as it only transfers accepted fee tokens to the fee aggregator which is a trusted address.
-  function withdrawFeeTokens() external {
-    address[] memory feeTokens = IFeeQuoter(s_dynamicConfig.feeQuoter).getFeeTokens();
+  /// @param feeTokens The fee tokens to withdraw.
+  /// @dev This function can be permissionless as it only transfers tokens to the fee aggregator which is a trusted address.
+  function withdrawFeeTokens(
```

Comment:
Should we create an interface for this in `shared`?

Reply:
Not sure shared is the correct place unless you want this interface to be implemented by all other products?

Reply:
Yeah, we're aiming to standardise this signature across all integrations

Reply:
Punting this to another PR